cobalt/cache/cache.cc

```diff
@@ -95,29 +95,30 @@ Cache* Cache::GetInstance() {
   return base::Singleton<Cache, base::LeakySingletonTraits<Cache>>::get();
 }
 
-bool Cache::Delete(disk_cache::ResourceType resource_type, uint32_t key) {
+bool Cache::Delete(cobalt::network::disk_cache::ResourceType resource_type,
```

Comment:
Here and below I believe you can substitute network::disk_cache for cobalt::network::disk_cache as we're in the cobalt namespace

Reply:
This looks like it wasn't changed?